CHANGELOG.md

```diff
@@ -4,49 +4,161 @@ All notable changes to this library will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this library adheres to Rust's notion of
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-The most recent changes are listed first.
 
-## [Unreleased]
+## Unreleased
 
 ### Added
+- `compact_formats.CompactTxIn`
+- `compact_formats.TxOut`
+- `service.PoolType`
 
-- Add debug logging to gRPC entry and exit points.
+### Changed
```

Comment:
@LarryRuane @nuttycom Shall we add which commit of the lightclient protocol files we are importing here?